frontend/src/views/producer/producer-modal/index.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import Modal, { Content } from '../../../components/modal';
+import Typography from '../../../components/typography';
+import { connect } from '../../../utils';
+import { withTheme } from 'styled-components';
+import Icon from 'antd/lib/icon';
+
+export default withTheme(
+  connect('store')(
+    class AppModal extends Component {
+      state = {
+        show: false,
+        handledSignalToShow: false,
+      };
+      constructor(props) {
+        super(props);
+        if (props.store.ui.auth.validateTokenFailed) {
+          this.state = { show: true };
+          window.setTimeout(() => {
+            try {
+              props.store.ui.auth.clearValidationError();
+            } catch (error) {
+              console.error(error);
+            }
+          }, 5000);
+        }
+      }
+      componentWillUpdate() {
+        // only show modal if its 0.5s pass waiting.
```

Comment:
If you are referring to the 5000 ms above, it's not 0.5s but 5s

Reply:
yes, im noting it